### ISSUE TITLE
Is this still working in 2021?

### ISSUE THREAD

**Issue by gentamatsui** [matzman666/OpenVR-InputEmulator#207]
Just wanted to see if anyone has this working. I tried the patch that someone made months ago but it didn't work for me. It's either I'm doing something wrong or it just broke with the new update.

**Comment by ghost** [comment]
Yes, I have it working. Had to uninstall then delete the folder and redownload from github. I then had to download a fresh dll. This is working on SteamVR 1.16.8 I use it to set the offsets for my tracker based guns. 

[driver_00vrinputemulator (2).zip](https://github.com/matzman666/OpenVR-InputEmulator/files/6096080/driver_00vrinputemulator.2.zip)

**Comment by gentamatsui** [comment]
> Yes, I have it working. Had to uninstall then delete the folder and redownload from github. I then had to download a fresh dll. This is working on SteamVR 1.16.8 I use it to set the offsets for my tracker based guns.
> 
> [driver_00vrinputemulator (2).zip](https://github.com/matzman666/OpenVR-InputEmulator/files/6096080/driver_00vrinputemulator.2.zip)

Wow thanks this worked for me. Cheers!

**Comment by Yersi88** [comment]
> Yes, I have it working. Had to uninstall then delete the folder and redownload from github. I then had to download a fresh dll. This is working on SteamVR 1.16.8 I use it to set the offsets for my tracker based guns.
> 
> [driver_00vrinputemulator (2).zip](https://github.com/matzman666/OpenVR-InputEmulator/files/6096080/driver_00vrinputemulator.2.zip)

Thank you good Sir

**Comment by ilimo** [comment]
https://ovrmc.dschadu.de/

Dschadu took over the project - follow that website

**Comment by Activusto** [comment]
> https://ovrmc.dschadu.de/
> 
> Dschadu took over the project - follow that website

This does not do or achieve the same thing as Input Emulator, and it never will as it's not the focus.
I have spoken to Dschadu myself about it.

**Comment by JstuffJr** [comment]
https://github.com/Yersi88/WMR-and-Vive-Tracker

Someone figured out how to edit vive tracker offset firmware with steamvr included terminal, so you can now do everyting completely native in SteamVR

**Comment by micmac253** [comment]
> Yes, I have it working. Had to uninstall then delete the folder and redownload from github. I then had to download a fresh dll. This is working on SteamVR 1.16.8 I use it to set the offsets for my tracker based guns.
> 
> [driver_00vrinputemulator (2).zip](https://github.com/matzman666/OpenVR-InputEmulator/files/6096080/driver_00vrinputemulator.2.zip)

ok and can you tell us where you put that dll?

**Comment by bowtar** [comment]
> > Yes, I have it working. Had to uninstall then delete the folder and redownload from github. I then had to download a fresh dll. This is working on SteamVR 1.16.8 I use it to set the offsets for my tracker based guns.
> > [driver_00vrinputemulator (2).zip](https://github.com/matzman666/OpenVR-InputEmulator/files/6096080/driver_00vrinputemulator.2.zip)
> 
> ok and can you tell us where you put that dll?

Should be in Steam\steamapps\common\SteamVR\drivers\00vrinputemulator\bin\win64
